gateway/fedimint-gateway-ui/src/lightning.rs

```diff
@@ -277,12 +311,192 @@ where
+                                                " — "
+                                                span { (format!("{:?}", tx.direction)) }
+                                            }
+
+                                            div style="font-size: 0.75rem; margin-top: 2px;" {
+                                                @let status_badge = match tx.status {
+                                                    PaymentStatus::Pending => html! { span class="badge bg-warning" { "⏳ Pending" } },
+                                                    PaymentStatus::Succeeded => html! { span class="badge bg-success" { "✅ Succeeded" } },
+                                                    PaymentStatus::Failed => html! { span class="badge bg-danger" { "❌ Failed" } },
+                                                };
+                                                (status_badge)
+                                            }
+                                        }
+
+                                        // Right: amount + timestamp
+                                        div style="text-align: right;" {
+                                            div style="font-weight: bold; font-size: 0.9rem;" {
+                                                (format!("{} sats", tx.amount.msats / 1000))
+                                            }
+                                            div style="font-size: 0.7rem; color: #6c757d;" {
+                                                @let timestamp = match Utc.timestamp_opt(tx.timestamp_secs as i64, 0) {
+                                                    LocalResult::Single(dt) => dt,
+                                                    _ => Utc.timestamp_opt(0, 0).unwrap(),
+                                                };
+                                                (timestamp.format("%Y-%m-%d %H:%M:%S").to_string())
+                                            }
+                                        }
+                                    }
+
+                                    // Optional hash/preimage, bottom row
+                                    @if let Some(hash) = &tx.payment_hash {
+                                        div style="font-family: monospace; font-size: 0.7rem; color: #6c757d; margin-top: 2px;" {
+                                            "Hash: " (hash.to_string())
+                                        }
+                                    }
+
+                                    @if let Some(preimage) = &tx.preimage {
+                                        div style="font-family: monospace; font-size: 0.7rem; color: #6c757d; margin-top: 1px;" {
+                                            "Preimage: " (preimage)
+                                        }
+                                    }
+
+                                    // Hover effect using inline JS (or move to CSS)
+                                    script {
```

Comment:
[nit] I believe this will make a new script and listener for each element. I think the css comment above might be a better approach but probably no biggie. Something like
```
.transaction-item:hover { background-color: #f8f9fa; }
```